lesson_2/mortgage_calc.py

```diff
@@ -0,0 +1,158 @@
+''' build a mortgage calculator that takes
+in loan amount, APR, and loan duration and returns the
+monthly payment assuming that interest is compounded monthly.'''
+
+# TODO: break up loan_calculator to be 10-15 lines only, make more functions of the small moving parts
+# TODO: make sure function names reference their side effect (ex display_intro_outro)
+import json
+with open('mortgage_messages.json', 'r') as file:
+    MESSAGES = json.load(file)
+
+def prompt(message):
+    print(f"==> {message}")
```

Comment:
I've seen this function in this project before... you may consider creating a `utils.py` file in the root of this project and importing that in all your other files going forward so you don't have to re-write this function each time ;)

Reply:
interesting, i'll have to look that up